packages/create-cloudflare/src/helpers/interactive.ts

```diff
@@ -11,6 +11,7 @@ const leftT = gray(shapes.leftT);
 export type TextOptions = {
 	renderSubmitted: (value: string) => string;
 	question: string;
+	initialValue?: string;
 	defaultValue: string;
 	acceptDefault: boolean | undefined; // must be specified, but can be undefined (≈ false)
```

Comment:
Actually this property could do with some documentation.
What I think this is really saying is "don't prompt the user, just use the default".
But this is now competing with `initialValue` for similar purpose, right?

Reply:
Do we even need this property once we have `initialValue`? I think if C3 has been instructed to use default values then the caller of the `textInput()` and similar functions would just set the `initialValue` to `defaultValue` and get the same result.